lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -1630,12 +1640,19 @@ fn test_monitor_update_fail_claim() {
 
 	*nodes[1].chain_monitor.update_ret.lock().unwrap() = Some(Err(ChannelMonitorUpdateErr::TemporaryFailure));
 	assert!(nodes[1].node.claim_funds(payment_preimage_1));
+	nodes[1].logger.assert_log("lightning::ln::channelmanager".to_string(), "Temporary failure claiming HTLC, treating as success: Failed to update ChannelMonitor".to_string(), 1);
```

Comment:
Is there any way to assert the state of the channel rather than examining the log?

Reply:
This line was only moved, not created. the log assert is nice IMO as it checks the specific error, not just that one happened, though the observable state is still checked in that no new message was generated.